_docker/drupal/drupal-filesystem/composer.json

```diff
@@ -81,7 +45,7 @@
         "drupal/console-core": "1.0.0-rc10",
         "drupal/console-en": "1.0.0-rc10",
         "drupal/drupal-extension": "^3.2",        
-        "drush/drush": "8.1.8",
+        "drush/drush": "^8.1.8",
```

Comment:
Can we lock the versions of the dependencies that we're using here rather than relying on "compatible with" or similar? 

Floating dependencies have bitten us once recently with https://issues.jboss.org/browse/DEVELOPER-4077. Because we build Docker images in each environment, it leaves it open for your PR to be built and tested against one version of a dependency, and then the version in production to be built against another.

cheers,

Rob